main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/appc/docker2aci/lib"
+	"github.com/coreos/rocket/cas"
+)
+
+const (
+	rocketDir = "/var/lib/rkt"
+)
+
+var flagImport = flag.Bool("import", false, "Import ACI images to the rocket store")
```

Comment:
while certainly convenient, I feel it's a little odd that this introduces a dependency on rocket (a specific implementation of the appc spec). I feel it's also more unix-y if the rocker import and docker2aci remain orthogonal. I personally wouldn't view it as much of a burden to chain this command with a command to import the generated ACI

(again, I'm not very familiar with the appc / rocket projects yet, just providing CLI design feedback as an excited user)